posthog/hogql/database/database.py

```diff
@@ -113,6 +115,7 @@ class Database(BaseModel):
     batch_export_log_entries: BatchExportLogEntriesTable = BatchExportLogEntriesTable()
     sessions: Union[SessionsTableV1, SessionsTableV2] = SessionsTableV1()
     heatmaps: HeatmapsTable = HeatmapsTable()
+    query_log: QueryLogTable = QueryLogTable()
```

Comment:
We should only be adding this class attribute when we've checked that we're allowed to. Likewise with the raw one too - we don't always check `posthog_tables` before trying to access the table